app/src/main/java/fr/free/nrw/commons/utils/SwipableCardView.java

```diff
@@ -43,6 +43,7 @@ private void interceptOnTouchListener() {
                     x1 = event.getX();
                     break;
                 case MotionEvent.ACTION_UP:
+                    v.performClick();
```

Comment:
Why has this been added?

Reply:
`'onTouch' lambda should call 'View#performClick' when a click is detected` lint warning is thrown

Reply:
@ashishkumar468 @neslihanturan Can confirm if this could cause any issues. Do you mind undoing this and creating a separate issue for this particular warning? :)

Apologies, for making you do change multiple times but my primary concern is not change anything that could cause other issues. Lint is helpful most of the times but it is not perfect. :)

Reply:
sure no problem